app_feup/lib/view/Pages/HomePageView.dart

```diff
@@ -14,7 +19,7 @@ class HomePageView extends StatelessWidget {
 
   Widget createActionButton(BuildContext context){
     return new FloatingActionButton(
-      onPressed: () => {}, //Add FAB functionality here
+      onPressed: () => getExams(), //Add FAB functionality here
```

Comment:
Before approving this pull request, the FAB shouldn't do anything - It's meant to allow adding widgets later on (maybe)

Reply:
Once again, only for testing, now removed